src/handlers/admin/createTemplate.ts

```diff
@@ -6,18 +6,36 @@ export default async function (
   auth: string,
   projectId: string,
   environmentId: string,
-  values: TemplateValues
-): Promise<TemplateResponse> {
+  values: TemplateValues | TemplateValues[]
+): Promise<TemplateResponse | TemplateResponse[]> {
   await checkAdminAccessUnwrapped(auth, projectId);
 
-  const template = await createTemplate({
-    id: values.id,
-    project_id: projectId,
-    environment_id: environmentId,
-    name: values.name,
-    rule: values.rule,
-    template: values.template,
-  });
+  let template: TemplateResponse | TemplateResponse[];
+  if (Array.isArray(values)) {
+    const newTemplates = await Promise.all(
+      values.map((v) =>
+        createTemplate({
```

Comment:
better to await here and do serial calls rather than make parallel calls to the db